study-builder/studies/brain/patches/prequal-v2.conf

```diff
@@ -0,0 +1,386 @@
+{
+  include required("../../snippets/activity-general-form.conf"),
+  "studyGuid": "cmi-brain",
+  "activityCode": "PREQUAL",
+  "versionTag": "v2",
+  "displayOrder": 1,
+  "writeOnce": true,
+  "maxInstancesPerUser": 1,
+  "excludeFromDisplay": true,
+  "allowUnauthenticated": true,
+  "timestamp": "2020-09-23T00:00:00Z",
```

Comment:
This timestamp is used as the starting point of v2 of the prequal. Shouldn't this be the time when we make the change in production instead of something hardcoded?

Reply:
yes it should be when we deploy to prod. For time being I added it. I will change it to use current system date in the patch code